packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -370,26 +369,19 @@ class DebuggerController extends DisposableController
     });
     _sortedScripts.value = scriptRefs;
 
-    // ignore: unawaited_futures
-    (_service as VmServiceWrapper)
-        .getClassList(isolateRef.id)
-        .then((Response response) {
-      final List classes = response.json['classes'];
-      final List<ClassRef> classRefs = classes
-          .map((json) => ClassRef.parse(json))
-          .cast<ClassRef>()
+    try {
+      final classList = await _service.getClassList(isolateRef.id);
+      final classes = classList.classes
```

Comment:
Shouldn't this end in a semi-colon? Perhaps I'm not reading it right.

You might want to make a copy of the list (`toList()`), to guard against that returned list not being mutable before you sort it.

Reply:
Yes, it looks like it should.

Reply:
No, it looks like there's more on the next line that wasn't changed.

Reply:
Ah, yeah. And it calls `toList()` at the end (after a `where`) so no need to make another copy.